java/arcs/core/storage/Handle.kt

```diff
@@ -44,36 +58,33 @@ interface Callbacks {
  * Implementations using this base are assumed to be based on a [CrdtModel] type. Concrete
  * subclasses should select a [CrdtData] and [CrdtOperation] type for the [Handle] when
  * implementing a concrete version of it, and expose only the consumer type [T].
+ *
+ * The base handle interface is at this layer to avoid circular dependencies.
  */
-abstract class Handle<Data : CrdtData, Op : CrdtOperation, T>(
+open class Handle<Data : CrdtData, Op : CrdtOperation, T>(
```

Comment:
Why the change from `abstract` to `open`? Is there a case where it would be meaningful to instantiate the base class?

Reply:
I use it in the storageproxy tests directly, since I don't want to depend on the subclasses impl.

Reply:
It's pretty easy to create a one-off subclass for testing from this, no?